server/pragma-checker/index.js

```diff
@@ -0,0 +1,67 @@
+/***** WARNING: ES5 code only here. Not transpiled! *****/
+
+/**
+ * External dependecies
+ */
+var error = require( 'chalk' ).bold.red;
+var info = require( 'chalk' ).bold.yellow;
+var startsWith = require( 'lodash/string/startsWith' );
+
+var PLUGIN_TITLE = 'PragmaChecker';
+var SSR_READY = '/** @ssr-ready **/';
+
+function PragmaCheckPlugin( options ) {
+	this.options = options || {};
+}
+
+function scanDependencies( module, compilation ) {
+	if ( ! module.dependencies ) {
+		return;
+	}
+
+	module.dependencies.forEach( function( dep ) {
+		// If the module is compiled through babel, we can be pretty sure it's our own module, not from npm.
+		if ( dep.module && /babel-loader/.test( dep.module.request ) &&
```

Comment:
Sure :) lodash's `includes` should be fast enough: https://github.com/lodash/lodash/blob/3.10.1/lodash.src.js#L6707